### DISCUSSION TITLE
Should I expected controller rumble to work with this setup?

### DISCUSSION THREAD

**Discussion by Siysrril** [LizardByte/community#25] [Sunshine] [answered]
Hi,
I run sunshine on this pc as host:

OS: Manjaro Linux
KERNEL: 5.16.11-2-MANJARO
CPU: AMD Ryzen 7 5800X 8-Core
GPU: AMD Radeon RX 5700 XT (NAVI10, DRM 3.44.0, 5.16.11-2-MANJARO, LLVM 13.0.1)
GPU DRIVER: 4.6 Mesa 21.3.7
RAM: 16 GB

My clients are Moonlight TV on a LG TV and a android phone, using a Xbox Series X controller, over Bluetooth in the TV and USB on android.

All works well except I'm not getting any rumble while streaming games.

Should Rumble on my controller work with that setup?

Regards.

**Comment by ReenigneArcher** [comment, marked as answer]
I may be mistaken, but I don't believe rumble over Bluetooth works on Android (and probably many other platforms).

https://www.nvidia.com/en-us/geforce/forums/gfn-tech-support/46/446974/xbox-one-bluetooth-controller-rumble-not-working/

**Reply by Siysrril** [reply]
I got controller rumble working after compiling and installing code from the nightly branch. However it only works for Non-Steam games, for example GOG games installed through Lutris.

Rumble doesn't work if I launch Steam big picture from any of my moonlight clients and then launch any steam game through it, however launching games outside of Steam it works. 

Any suggestions?

**Reply by ReenigneArcher** [reply]
Steam is taking over the controller inputs. I'd look through the steam settings and try tweaking how it handles the gamepads.

**Reply by Siysrril** [reply]
That was it!!
Steam was recognizing the controller as 360 controller without rumble. Had to uninstall a couple drivers, an then disabled steam input per game I wanted to stream. And now it's perfect!!!

Gotta say I'm Impressed with the smoothness that sunshine + moonlight can achieve, once everything is setup properly.

Also went back to test a Dualshock 4 controller over Bluetooth and rumble work flawless. However on a Xbox series x controller over Bluetooth it doesn't work, I believe this is a issue specific of the series x controller, I remember that not even steam link supports rumble over Bluetooth for that controller (something about an api not available from Microsoft).
